examples/ota-requestor-app/esp32/main/main.cpp

```diff
@@ -73,6 +74,33 @@ static void InitServer(intptr_t context)
 
 } // namespace
 
+// Test mode operation
+void OnStartDelayTimerHandler(Layer * systemLayer, void * appState)
```

Comment:
nit: This function name doesn't really reflect what it is actually doing

Reply:
Done